app/controllers/forms_controller.rb

```diff
@@ -58,7 +69,7 @@ def update
       @form.transaction do
         # @form.updated_by = current_user
         @form.form_questions.destroy_all
-        @form.form_questions = create_form_questions(params[:form][:linked_questions], params[:form][:linked_response_sets])
+        @form.form_questions = create_form_questions
         # When we assign update_successful, it is the last expression in the block
         # That means, if the form fails to update, this block will return false,
         # which will cause the transaction to rollback.
```

Comment:
Possibly get rid of some of comments to resolve rubocop error (not sure if they are being counted in the line count):
```
Offenses:

app/controllers/forms_controller.rb:1:1: C: Class has too many lines. [108/100]
class FormsController < ApplicationController ...
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

230 files inspected, 1 offense detected
```